src/Core/Managed/Shared/Extensibility/Implementation/TelemetryConfigurationFactory.cs

```diff
@@ -83,15 +83,20 @@ public virtual void Initialize(TelemetryConfiguration configuration, TelemetryMo
                     configuration.InstrumentationKey = environmentIKey;
                 }
 
-                // Creating the default channel if no channel configuration supplied
-                configuration.TelemetryChannel = configuration.TelemetryChannel ?? new InMemoryChannel();
```

Comment:
Yes it does, I expect some things to break if there is no default channel created somewhere...